lib/src/model/package_graph.dart

```diff
@@ -72,7 +72,7 @@ class PackageGraph {
     await Future.wait(precacheLocalDocs());
     _localDocumentationBuilt = true;
 
-    // Scan all model elements to insure that interceptor and other special
+    // Traverse all can all model elements to insure that interceptor and other special
```

Comment:
I can't parse this sentence... typo?

Reply:
yes.  This was a draft PR, I didn't realize you were going to jump on it right away.  Hadn't done my final typo scan yet :-)